bitnami/redis/templates/master/service.yaml

```diff
@@ -45,7 +45,11 @@ spec:
   externalIPs: {{- include "common.tplvalues.render" (dict "value" .Values.master.service.externalIPs "context" $) | nindent 4 }}
   {{- end }}
   ports:
+      {{- if .Values.master.service.portNames.redis }}
+    - name: {{ .Values.master.service.portNames.redis }}
+      {{- else }}
     - name: tcp-redis
+      {{- end }}
```

Comment:
```suggestion
    - name: {{ .Values.master.service.portNames.redis }}
```

Reply:
Done - also updated the default in values.yaml